### PR TITLE
Edits chptr 6

### PR DESCRIPTION
**Pull request by katygregg**
Good stuff! :)

### CODE REVIEW

File: 06-read-write-plot.Rmd [hunk 1]
```diff
@@ -125,7 +125,7 @@ Although the quality is not as spatially consistent as many official datasets, O
 Further examples of **osmdata** in action are provided in Chapters \@ref(transport) and  \@ref(location).
 <!-- Todo: Replace the above with "Chapters \@ref(location) and \@ref(gis)." when gis is online (RL) -->
 
-Finally, R packages might contain or just consist of spatial data (e.g., package **spData**).
+Finally, R packages might contain or just consist of spatial data (e.g. package **spData**).
```
Comment (Robinlovelace) [inline 06-read-write-plot.Rmd]: I've just spotted another issue - 'package ...' is not good English. `(for example the **spData** package)` would be better. Will change it now. Thanks for the fix!

File: 06-read-write-plot.Rmd [hunk 2]
```diff
@@ -144,27 +144,27 @@ Find more information on getting data using R packages in [section 5.5](https://
 Spatial data is usually stored as files or in spatial databases. 
 File-based data formats can contain either vector or raster data, while spatial databases could store both.
 Historically, GIS file formats were developed by mapping agencies and software companies. <!--examples-->
-Exchanging spatial data between different software packages each coming with its own format was troublesome in the beginning.
+Exchanging spatial data between different software packages, each coming with its own format, was troublesome in the beginning.
```
Comment (Robinlovelace) [inline 06-read-write-plot.Rmd]: In the beginning, there was the ⚡️  Shapefile ⚡️

**Comment by Robinlovelace** [comment]
Thanks loads!

**Comment by Nowosad** [comment]
Thanks @katygregg !

**Pull request merged**